doc/getting_started/index.rst

```diff
@@ -128,77 +95,36 @@ Next, install additional Python packages required by Zephyr in a shell or
    # macOS and Windows
    pip3 install -r zephyr/scripts/requirements.txt
 
-Some notes on pip's ``--user`` option:
-
-- Installing with ``--user`` is the default behavior on Debian-based
-  distributions and is generally recommended on Linux to avoid conflicts with
-  Python packages installed using the system package manager.
-
-- On Linux, verify the Python user install directory ``~/.local/bin`` is at the front of
-  your PATH environment variable, otherwise installed packages won't be
-  found.
-
-- On macOS, Homebrew disables the ``--user`` flag\ [#homebrew_user]_.
-
-- On Windows using ``cmd.exe``, although it's possible to use the ``--user``
-  flag, it makes it harder for the command prompt to find executables installed
-  by pip.
-
-
 Set Up a Toolchain
 ******************
 
-.. note::
+Zephyr binaries are compiled and linked by a *toolchain*\
+[#tools_native_posix]_. You now need to *install* and *configure* a toolchain.
+Toolchains are *installed* in the usual ways you get programs: with installer
+programs or system package managers, by downloading and extracting a zip
+archive, etc.
 
-   On Linux, you can skip this step if you installed the :ref:`Zephyr SDK
-   <zephyr_sdk>`, which includes toolchains for all supported Zephyr
-   architectures.
+You *configure* the toolchain by setting environment variables. You need to set
```

Comment:
I think you mean "select" the toolchain instead of "configure" the toolchain. The latter sounds like changing the way the toolchain behaves.

Reply:
I don't think so. ZEPHYR_TOOLCHAIN_VARIANT could maybe be described as "selecting" a toolchain, but the other variables which set its location are surely configuration options. For example, I keep several old Zephyr SDKs installed on my system, and configure the one which is currently in use based on how old the Zephyr tree I'm building against is.

Reply:
I'm not a native speaker (@dbkinder help?) but I understand that choosing whether to use SDK 0.9.5 or SDK 0.10.0 is configuring the Zephyr build, not "configuring the toolchain". Which one by the way? Both?
Configuring one toolchain would be changing its behaviour in some way and while possible I haven't found any example of that in the current documentation; it belongs to toolchain-specific documentation.

Reply:
I still disagree FWIW.